CDCgov/prime-reportstream
# 9471 - Remove hyperlink on file download where date has expired.

This PR ...

If a file has expired and a user clicks on the file to download, the file does not download and a console error is thrown which notifies Dev Ops.
Do not allow the user to download the file when the date has expired.

Test Steps:
1. Log into RS
2. Navigate to Daily Data
3. If the date has expired in the row, the file should not be a hyperlink.

## Changes

<img width="1426" alt="Screenshot 2023-05-19 at 9 14 41 AM" src="https://github.com/CDCgov/prime-reportstream/assets/102491809/87c78555-0089-426c-8ad1-3edede06a949">


## Checklist

### Testing

## Linked Issues
- Fixes #9471
